### PR TITLE
Bump dependencies of k8s to 0.19.2

### PR DESCRIPTION
**Pull request by rcarrillocruz**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -8,30 +8,23 @@ require (
...
-	sigs.k8s.io/controller-runtime v0.5.2
-)
-
-replace (
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.18.6
-	k8s.io/client-go => k8s.io/client-go v0.18.6
-	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.18.6
-	sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.0.0-20200623140740-add0b6444f50
+	k8s.io/api v0.19.0
+	k8s.io/apimachinery v0.19.0
```
Comment (aojea) [inline go.mod:25]: 1.9.3 was releases 5 days ago, 
https://github.com/kubernetes/kubernetes/releases/tag/v1.19.3

Comment (rcarrillocruz) [reply go.mod]: Check it out other openshift operators, they are either at 0.19 or 0.19.2 .

**Comment by rcarrillocruz** [comment]
We need to land https://github.com/openshift/release/pull/12907 , unit tests are running on golang 1.13 thus  the crypto failures as those symbols naming changed.

**Comment by danwinship** [comment]
/lgtm
if it passes enough tests to merge...

**Pull request merged**